UiPath.Ipc.net6.0-windows.received.txt

```diff
@@ -0,0 +1,264 @@
+[assembly: System.Reflection.AssemblyMetadata("RepositoryUrl", "https://github.com/UiPath/coreipc.git")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Playground")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("UiPath.CoreIpc.BackCompat")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("UiPath.CoreIpc.Tests")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("UiPath.Ipc.TV")]
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("UiPath.Ipc.Tests")]
+[assembly: System.Runtime.Versioning.SupportedOSPlatform("Windows7.0")]
+[assembly: System.Runtime.Versioning.TargetFramework(".NETCoreApp,Version=v6.0", FrameworkDisplayName=".NET 6.0")]
+[assembly: System.Runtime.Versioning.TargetPlatform("Windows7.0")]
+namespace UiPath.Ipc
+{
+    public readonly struct CallInfo
+    {
+        public CallInfo(bool newConnection, System.Reflection.MethodInfo method, object?[] arguments) { }
+        public object?[] Arguments { get; }
+        public System.Reflection.MethodInfo Method { get; }
+        public bool NewConnection { get; }
+    }
+    public sealed class ClientConfig : UiPath.Ipc.EndpointConfig, System.IEquatable<UiPath.Ipc.ClientConfig>
+    {
+        public ClientConfig() { }
+        public string DebugName { get; set; }
+        public UiPath.Ipc.ISerializer? Serializer { get; set; }
+        public System.Func<UiPath.Ipc.CallInfo, System.Threading.CancellationToken, System.Threading.Tasks.Task>? BeforeCall { get; init; }
+        public System.Func<System.Threading.CancellationToken, System.Threading.Tasks.Task>? BeforeConnect { get; init; }
+        public UiPath.Ipc.EndpointCollection? Callbacks { get; init; }
+        public Microsoft.Extensions.Logging.ILogger? Logger { get; init; }
+        public System.Threading.Tasks.TaskScheduler? Scheduler { get; init; }
+        public System.IServiceProvider? ServiceProvider { get; init; }
+    }
+    public abstract class ClientTransport : System.IEquatable<UiPath.Ipc.ClientTransport>
```

Comment:
All those who implement `IEquatable` in the public surface are records, i.e.: [`ClientTransport`](https://github.com/UiPath/coreipc/blob/6b263e6b7868a6a2178341616492fdc61c5cd1b8/src/UiPath.CoreIpc/Config/ClientTransport.cs#L3)
![image](https://github.com/user-attachments/assets/4bf068d5-cdb4-459c-825f-81a1bae029e9)

It was valuable to make them all records; it wasn't a fluke. If it's too cumbersome for this or any other reason, I can rethink it, but with great care, because they're immutable, and I used `with` to alter them through construction virtually.